api/base/exceptions.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.exceptions import APIException
+from rest_framework import status
+
+#Custom Exceptions the Django Rest Framework does not support
+
+class Gone(APIException):
+    status_code = status.HTTP_410_GONE
+    default_detail = 'Gone: The requested resource is no longer available.'
```

Comment:
Remove `"Gone: "`.  Looking at the other exceptions in `rest_framework.exceptions`, none of them include the text status.